src/hotspot/share/prims/jvmtiTagMap.cpp

```diff
@@ -2225,6 +2227,138 @@ class JNILocalRootsClosure : public OopClosure {
   virtual void do_oop(narrowOop* obj_p) { ShouldNotReachHere(); }
 };
 
+// Helper class to collect/report stack roots.
+class StackRootCollector {
```

Comment:
We discussed privately about the following renamings:
 - `StackRootCollector` => `StackRefCollector`
 - `collect_stack_roots` => `collect_stack_refs`
 - `collect_vthread_stack_roots` => `collect_vthread_stack_refs`

Reply:
done